README.md

```diff
@@ -458,20 +458,74 @@ The latter script will print a section for the `gatk-sv.toml` `[images]` section
 
 ### Dev installation
 
-Assuming `python3` points to `python3.10`, set up a virtual environment:
+0. _Optional_ Assuming `python3` points to `python3.10`, set up a virtual environment:
 
-```sh
-python3 -m pip install virtualenv
-virtualenv venv
-source venv/bin/activate
-```
+    ```sh
+    python3 -m pip install virtualenv
+    virtualenv venv
+    source venv/bin/activate
+    ```
+
+1. Install the package, and the development environment:
+
+    ```bash
+    pip install -e .
+    pip install -r requirements-dev.txt
+    ```
+
+
+2. Generate a `dev.toml` config file to mimic the analysis-runner environment. 
+
+- You can generate this with the analysis runner for example 
+
+    ```bash
+    analysis-runner config --dataset thousand-genomes --access-level test --output-dir "1kg" --config configs/genome.toml --config configs/thousand-genomes.toml
+    ``` 
+- Alternatively, you can create your own `dev.toml` file with the config that’s logged in Hail Batch. 
+
+3. In `dev.toml`, ensure the following parameter is set 
+
+    ```toml
+    [hail]
```

Comment:
Per https://centrepopgen.slack.com/archives/C030X7WGFCL/p1678745923726729, does this need to be updated?